jni/tests/test_util.h

```diff
@@ -150,6 +150,16 @@ namespace test_util {
 
     float RandomFloat(float min, float max);
 
+    // Read vector file formats
+    void load_data(char* filename, float*& data, unsigned& num, unsigned& dim);
+
+    // asign data into vector
```

Comment:
nit: asign -> assign. Also, can we add more detail about how this function should be used in the comment?